packages/xo-web/src/common/intl/messages.js

```diff
@@ -570,6 +570,7 @@ const messages = {
   reportRecipients: 'Report recipients',
   reportWhen: 'Report when',
   concurrency: 'Concurrency',
+  mergeBackupsSynchronously: 'Merge backups synchronously',
```

Comment:
```suggestion
  mergeBackupsSynchronously: 'Merge backups synchronously. This will use more resources on the backup thread, but ensure there is no locking error when chaining multiple backup jobs on the same remote',
```

Reply:
Done